src/topology/algebra/group.lean

```diff
@@ -167,12 +167,13 @@ section quotient_topological_group
 variables [topological_space α] [group α] [topological_group α] (N : set α) [normal_subgroup N]
 
 @[to_additive]
-instance : topological_space (quotient_group.quotient N) :=
+instance {α : Type u} [group α] [topological_space α] (N : set α) [normal_subgroup N] :
```

Comment:
What happened here? It seems all these arguments are listed as `variables` immediately above.

Reply:
Presumably it's avoiding the `[topological_group α]` instance?

Reply:
Yes, the `[topological_group α]` is not necessary here.